README.md

```diff
@@ -3201,6 +3201,10 @@ For compatibility reasons, these NPM packages are embedded into bun’s binary a
 
 Estimated: 30-90 minutes :(
 
+### Prerequisites 
+
+- On all systems, [Docker](https://www.docker.com) is required. If using WSL on Windows, it is recommended to use [Docker Desktop](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers) for its WSL2 integration.
```

Comment:
This isn't true. Docker is only required if using the devcontainer

Reply:
my bad, thanks for the correction! Fixing.

Reply:
Fixed, with some organization improvements too.